ramls/hridsetting.json

```diff
@@ -14,6 +14,11 @@
       "type": "integer",
       "minimum": 1,
       "maximum": 99999999999
+    },
+    "retainLeadingZeroes" : {
+      "type": "boolean",
+      "description": "If true, leading zeros should be applied for HRIDs",
+      "default": true
```

Comment:
A new property has been introduced, this should be reflected in a the minor version of the interface (both in the module descriptor and RAML)

Reply:
Completed. Thank you.